CMakeLists.txt

```diff
@@ -220,7 +220,7 @@ if(BUILD_PYTHON)
   )
 endif()
 
-set(NVFUSER_CODEGEN ${PROJECT_NAME}_codegen)
+set(NVFUSER_CODEGEN ${PROJECT_NAME}_codegenx) # the "x" suffix is to make sure there is no naming conflict with the old libnvfuser_codegen.so in torch/lib
```

Comment:
nitpick: Can we have a proper name here instead of a `x` suffix. :stuck_out_tongue: